xml/containers-build.xml

```diff
@@ -0,0 +1,171 @@
+    <link xlink:href="https://build.opensuse.org/image_templates">container
+    image templates</link> can be modified to specific needs and are the
+    easiest way to create your own container branch. Container images can
+    either be built with <emphasis role="strong">native Docker tools</emphasis>
+    - based on an existing image using a Dockerfile. Or they can be built from
+    scratch using the <emphasis role="strong">&kiwi; imaging
+    solution</emphasis>.
+   </para>
+   <para>
+    Instructions on how to build images on OBS can be found
+    <link xlink:href="https://openbuildservice.org/2018/05/09/container-building-and-distribution/">here</link>.
+   </para>
+  </sect2>
+
+  <sect2 xml:id="id-kiwi">
+   <title>&kiwi;</title>
+   <para>
+    <emphasis role="strong">KIWI Next Generation</emphasis> is a multi purpose
+    imaging solution. Beside container images, classical installation ISOs and
+    images for virtual machines &kiwi; can also build images that boot via PXE
+    or Vagrant boxes. The basis is an image XML description. Such a description
+    is represented by a directory which includes at least one
+    <literal>config.xml</literal> or <literal>.kiwi</literal> file and may as
+    well include other files like scripts or configuration data. The image
+    creation process with &kiwi; is automated and does not require any user
+    interaction. Any information required for the image creation process is
+    provided by the primary configuration file: <literal>config.xml</literal>.
+    In addition, the image can optionally be customized using the
+    <literal>config.sh</literal> and <literal>images.sh</literal> scripts.
+   </para>
+   <note>
+    <para>
+     It is important to distinguish between KIWI NG (currently version 9.20.9)
+     and it’s unmaintained legacy versions (7.x.x or older), now called
+     <link xlink:href="https://doc.opensuse.org/projects/kiwi/doc/">KIWI
+     Legacy</link>.
+    </para>
+   </note>
+   <para>
+    For specific information on how to install &kiwi; and use it to build
+    images, see the <link xlink:href="http://osinside.github.io/kiwi/">KIWI
+    documentation</link>. A collection of example image descriptions can be
+    found on the
+    <link xlink:href="https://github.com/OSInside/kiwi-descriptions">KIWI
+    GitHub repository</link>.
+   </para>
+   <para>
+    You can also find out more about &kiwi; through it’s man pages by
+    installing the <literal>kiwi-man-pages</literal> package and running the
+    <literal>man kiwi</literal> command.
+   </para>
+  </sect2>
+ </sect1>
+ <sect1 xml:id="id-building-official-slea-images">
+  <title>Building Official &slea; Images</title>
+
+  <para>
+   Only images built via the Internal Build Service,
+   <emphasis role="strong">build.suse.de</emphasis>, can be official.
+  </para>
+
+  <para>
+   There are no official SLE container images on
+   <link xlink:href="https://build.opensuse.org">https://build.opensuse.org</link>
```

Comment:
No need for the link text, if it's just the URL again -- this can be made a self-closing `<tag/>`.